omod-1.12/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_12/OrderGroupController1_12Test.java

```diff
@@ -0,0 +1,141 @@
+			"      \"dateActivated\": \"2012-09-19\",\n" + 
+			"      \"scheduledDate\": \"2012-09-19\",\n" + 
+			"      \"patient\": \"19887889-7110-46a7-a9f0-ebf1eb4bbd48\",\n" + 
+			"      \"concept\": \"985ff1a2-c2ef-49fd-836f-8a1d936d9ef9\",\n" + 
+			"      \"careSetting\": \"6f0c9a92-6f24-11e3-af88-005056821db0\",\n" + 
+			"      \"autoExpireDate\": \"2012-09-21\",\n" + 
+			"      \"orderer\": \"c2299800-cca9-11e0-9572-0800200c9a66\",\n" + 
+			"      \"previousOrder\": \"\",\n" + 
+			"      \"urgency\": \"ROUTINE\",\n" + 
+			"      \"orderReason\": \"\",\n" + 
+			"      \"orderReasonNonCoded\": \"string\",\n" + 
+			"      \"instructions\": \"string\",\n" + 
+			"      \"commentToFulfiller\": \"string\"\n" + 
+			"    }\n" + 
+			      "," +
+			"    {\n" + 
+			"      \"encounter\": \"33d70956-b359-452a-b3da-b69c8ab459ce\",\n" + 
+			"      \"orderType\": \"2d3fb1d0-ae06-22e3-a5e2-0140211c9a66\",\n" + 
+			"      \"type\": \"order\",\n" + 
+			"      \"action\": \"NEW\",\n" + 
+			"      \"accessionNumber\": \"string\",\n" + 
+			"      \"dateActivated\": \"2012-09-10\",\n" + 
+			"      \"scheduledDate\": \"2012-09-10\",\n" + 
+			"      \"patient\": \"19887889-7110-46a7-a9f0-ebf1eb4bbd48\",\n" + 
+			"      \"concept\": \"985ff1a2-c2ef-49fd-836f-8a1d936d9ef9\",\n" + 
+			"      \"careSetting\": \"6f0c9a92-6f24-11e3-af88-005056821db0\",\n" + 
+			"      \"autoExpireDate\": \"2012-09-21\",\n" + 
+			"      \"orderer\": \"c2299800-cca9-11e0-9572-0800200c9a66\",\n" + 
+			"      \"previousOrder\": \"\",\n" + 
+			"      \"urgency\": \"ROUTINE\",\n" + 
+			"      \"orderReason\": \"\",\n" + 
+			"      \"orderReasonNonCoded\": \"string\",\n" + 
+			"      \"instructions\": \"string\",\n" + 
+			"      \"commentToFulfiller\": \"string\"\n" + 
+			"    }\n" + 
+			"  ],\n" + 
+			"  \"orderSet\": \"2d3fb1d0-ae06-22e3-a5e2-0140211c1001\"\n" + 
+			"}";
+	
+	@Before
+	public void init() throws Exception {
+		orderService = Context.getOrderService();
+		executeDataSet(RestTestConstants1_12.ORDER_GROUP_TEST_DATA_SET);
+	}
+	@Override
+	public String getURI() {
+
+		return "orderGroup";
+	}
+
+	@Override
+	public String getUuid() {
+		return RestTestConstants1_12.ORDER_GROUP_UUID;
+	}
+
+	@Override
+	public long getAllCount()  {
+		return 0;
+	}
+
+	@Test
+	public void shouldCreateNewOrderGroup() throws Exception{
+		
+				MockHttpServletRequest req = newPostRequest(getURI(), json);
+				SimpleObject result = deserialize(handle(req));
+				Assert.assertEquals(patient_uuid, Util.getByPath(result, "patient/uuid"));
+				Assert.assertEquals(encounter_uuid,Util.getByPath(result, "encounter/uuid"));
+				Assert.assertEquals(orderSet_uuid,Util.getByPath(result, "orderSet/uuid"));
+				Assert.assertEquals(display, Util.getByPath(result, "display"));
+				
+				String uuid =  (String) PropertyUtils.getProperty(result, "uuid");
+				Assert.assertEquals(2, orderService.getOrderGroupByUuid(uuid).getOrders().size());			
```

Comment:
How do you tell that the size was less than this before making the REST call?

Reply:
alryt. i will update this after we recover from the  data-center shutdown , other-wse every Pr i make today fails , because of failure for travis to connect to the openmrs maven repo

Reply:
Actually so far what i did, the orderGroup is not updatable, you can only create it with a given number of orders, but cant be updated. so Is an orde group supposed to be updatable ?? say adding in more orders??

Reply:
for this test , its for creating a purely new ordergroup with 2 orders . So the orderGroup doesnt exist before the rest call

Reply:
but i have aded a test showing adding more orders to an existing OrderGroup